.gitignore

```diff
@@ -14,3 +14,8 @@
 /denoDist
 /npm
 /deno
+/build
+.docusaurus
+.cache-loader
+.DS_Store
+npm-debug.log*
```

Comment:
Both `.DS_Store` and `npm-debug.log` are not docusaurus specific and should be ignored in the global `.gitignore` of the developer.

Reply:
I would like to have it here but okay.